applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -154,7 +154,9 @@ static void get_codec_info(const struct bt_audio_codec_cfg *codec,
 	}
 
 	ret = bt_audio_codec_cfg_get_chan_allocation(codec, &codec_info->chan_allocation);
-	if (ret) {
+	if (ret == -ENODATA) {
+		codec_info->chan_allocation = 0;
```

Comment:
Add a comment that we are reverting to mono if no Channel_allocation is available